pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -47,25 +47,28 @@ type registryReconcilerFactory struct {
 	Lister               operatorlister.OperatorLister
 	OpClient             operatorclient.ClientInterface
 	ConfigMapServerImage string
+	debugLogging         bool
 }
 
 // ReconcilerForSource returns a RegistryReconciler based on the configuration of the given CatalogSource.
-func (r *registryReconcilerFactory) ReconcilerForSource(source *v1alpha1.CatalogSource) RegistryReconciler {
+func (r *registryReconcilerFactory) ReconcilerForSource(source *v1alpha1.CatalogSource, debugLogging bool) RegistryReconciler {
```

Comment:
Since `debugLogging` is already a member of `registryReconcilerFactory`, I don't think we need this parameter.

Reply:
Oops, I double plumbed.